.github/workflows/version-auto-bump.yml

```diff
@@ -0,0 +1,121 @@
+---
+name: Version Auto Bump
+
+on:
+  release:
+    types: [published]
+
```

Comment:
I believe this workflow could be tested if added the following and pushed it in a new commit. Keeping the branch publish disabled in your comments.
```yaml
on:
  release:
    types: [published]
  push:
```

Reply:
I added the manual trigger since the `tag` needs to be supplied somehow. For testing it'll be sent via input in manual trigger.